hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcServer.java

```diff
@@ -2357,6 +2357,11 @@ public synchronized void start() {
     started = true;
   }
 
+  @Override
+  public void refreshAuthManager(PolicyProvider pp) {
+    // noop
```

Comment:
Can we not call the static method from here?

Reply:
`RpcServer` is IA.private, and #refreshAuthManager shouldn't be called elsewhere, do we still need the concrete impl